meta/meta.go

```diff
@@ -141,18 +141,20 @@ func (m *Meta) parseTableID(key string) (int64, error) {
 }
 
 // GenAutoTableID adds step to the auto ID of the table and returns the sum.
-func (m *Meta) GenAutoTableID(dbID int64, tableID int64, step int64) (int64, error) {
+func (m *Meta) GenAutoTableID(originalDBID, dbID, tableID, step int64) (int64, error) {
+	// Check if DB exists.
```

Comment:
Do we also need to check if old db exists or not ?

Reply:
The old db doesn't need to exist, it can be dropped.